package.json

```diff
@@ -64,6 +64,8 @@
     "jstz": "~1.0.7",
     "lodash": "~4.15.0",
     "multiselect": "~0.9.12",
+    "patternfly": "^3.29.5",
```

Comment:
Is that required ? Afair it should be pulled automatically?

Reply:
It's a [peer dependency](https://github.com/patternfly/patternfly-react/blob/master/package.json#L64) so it doesn't come along for the ride. We have to add it manually.